phpunit.xml.dist

```diff
@@ -10,6 +10,7 @@
     <php>
         <ini name="error_reporting" value="-1" />
         <server name="KERNEL_DIR" value="app/" />
+        <env name="DATABASE_URL" value="sqlite://%kernel.project_dir%/var/data/blog_test.sqlite"/>
```

Comment:
@javiereguiluz, compiled container doesn't interpolate parameters in environment variables.  See https://github.com/symfony/symfony/blob/v3.3.0/src/Symfony/Component/DependencyInjection/Container.php#L442